crates/astria-composer/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "astria-composer"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+astria-sequencer = { path = "../astria-sequencer" }
+astria-sequencer-client = { path = "../astria-sequencer-client" }
+async-trait = { workspace = true }
+color-eyre = { workspace = true }
+ed25519-consensus = { workspace = true }
+figment = { workspace = true, features = ["toml", "env", "test"] }
+tokio = { workspace = true, features = ["full"] }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+clap = { workspace = true, features = ["derive"] }
+thiserror = { workspace = true }
+tracing = { workspace = true, features = ["attributes"] }
+tracing-subscriber = { workspace = true, features = ["env-filter", "json"] }
+reqwest = { workspace = true, features = ["json"] }
+is-terminal = { workspace = true }
+axum = { workspace = true }
+hyper = { workspace = true }
+ethers = { version = "2.0.7", features = ["ws"] }
+tokio-stream = "0.1.14"
+hex = { workspace = true }
+futures.workspace = true
+tendermint-rpc = { workspace = true }
+rand = "0.8.5"
```

Comment:
extreme nit but can we separate the deps that are `workspace = true` vs not?

Reply:
I went through all toml files and cleaned them up, ordering alphabetically and grouping local vs workspace